resources/production/chainspec.toml

```diff
@@ -1,4 +1,12 @@
-[genesis]
+[protocol]
```

Comment:
Why does the production chainspec does not follow the pattern where the chainspec is stored in the directory with a name that is a protocol version?

Reply:
I'd like to compare notes with @sacherjj before making changes like that.  Once we decide how to proceed, we might well nest the configs and chainspecs in version folders.  But there's not really a need - it's more that the install process should adhere to that naming pattern.